testsuite/features/support/namespaces/system.rb

```diff
@@ -12,11 +12,7 @@ def initialize(api_test)
     @search = NamespaceSystemSearch.new(api_test)
   end
 
-  attr_reader :config
-  attr_reader :custominfo
-  attr_reader :provisioning
-  attr_reader :scap
-  attr_reader :search
+  attr_reader :config, :custominfo, :provisioning, :scap, :search
```

Comment:
I prefer the old style, although it occupies more space in the file.

Reply:
I prefer to keep this Cop, and this style, I made some research of code examples and most of them use this style.